ui/src/demos/language-modeling/Predictions.tsx

```diff
@@ -38,18 +38,25 @@ interface SingleResultProps {
 const SingleResult = ({ pred, input }: SingleResultProps) => {
     const form = useContext(Form);
     const completion = cleanTopTokensForDisplay(pred);
+    const displayCompletion =
+        completion.slice(-1) === '.' || completion.slice(-1) === '↵'
+            ? completion
+            : completion.concat(' ...');
```

Comment:
```suggestion
            : completion.concat(' …');
```

This uses the utf-8 ellipsis instead of three periods. This is nicer for accessibility clients, search engines and is a single character.